lib/resources/crontab.rb

```diff
@@ -25,21 +24,26 @@ class Crontab < Inspec.resource(1)
       describe crontab.where { command =~ /a partial command string/ } do
         its('entries.length') { should cmp 1 }
       end
+
+      describe crontab('/etc/cron.d/some_crontab') do
+        its('commands') { should include '/path/to/some/script' }
+      end
     "
 
     attr_reader :params
 
     include CommentParser
 
-    def initialize(user = nil)
-      @user   = user
+    def initialize(destination = nil)
```

Comment:
I am a little bit unsure about the overloading of the destination. It can be a user name or a path. Wouldn't it be cleaner to use something like:
`describe crontab({path: '/etc/cron.d/some_crontab'})` and `describe crontab({user: 'chris'})`

Reply:
Yes will do!